src/crystal/system/win32/registry.cr

```diff
@@ -0,0 +1,100 @@
+require "c/winreg"
+require "c/regapix"
+
+module Crystal::System::Registry
```

Comment:
Should `Windows` apparent as a namespace here at all? I'd prefer `Crystal::System::Windows::Registry` or even `Crystal::System::WindowsRegistry`. I don't think there's a "registry" in unix, right?

Reply:
Yes, this is exclusive to windows. But I don't see a strong reason to apply a prefix or namespace. It's just an internal name, there's not going to be any ambiguity with some other kind of registry.